src/App.js

```diff
@@ -14,11 +15,88 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGoods: [],
+  }
+
+  addOrRemoveSelectedGood = (event, product) => {
+    const { selectedGoods } = this.state;
+    const newSelectedGoods = [...selectedGoods];
+    const index = newSelectedGoods.indexOf(product);
+
+    if (index === -1) {
+      newSelectedGoods.push(product);
+    } else {
+      newSelectedGoods.splice(index, 1);
+    }
+
+    this.setState({
+      selectedGoods: newSelectedGoods,
+    });
+  }
+
+  clearSelectedGoods = () => {
+    this.setState({
+      selectedGoods: [],
+    });
+  }
+
+  printHeader = () => {
+    const { selectedGoods } = this.state;
+
+    if (selectedGoods.length === 0) {
+      return '';
+    }
```

Comment:
You don't need this. If your array is empty (length === 0), it will just return nothing and the string will be empty